rules/CodingStyle/ClassNameImport/AliasUsesResolver.php

```diff
@@ -45,9 +46,14 @@ public function resolveFromStmts(array $stmts): array
         $aliasedUses = [];
 
         $this->useImportsTraverser->traverserStmts($stmts, static function (
+            int $useType,
```

Comment:
Something like this:

```suggestion
           /** @param Use_::TYPE_* $useType */
                $this->useImportsTraverser->traverserStmts($stmts, static function (
            int $useType,
```